akka-http-core/src/main/scala/akka/http/impl/engine/client/PoolConductor.scala

```diff
@@ -85,7 +90,11 @@ private object PoolConductor {
       Ports(retryMerge.in(0), retrySplit.in, route.outArray.toList)
     }
 
-  private case class SwitchCommand(rc: RequestContext, slotIx: Int)
+  sealed trait SlotCommand
+  case class DispatchCommand(rc: RequestContext) extends SlotCommand
+  case object SlotShouldConnectCommand extends SlotCommand
```

Comment:
Maybe `ConnectEagerlyCommand`? It would fit well with `SlotEvent.ConnectedEagerly`